function/pick/picker.go

```diff
@@ -20,13 +20,13 @@ type Picker struct {
 }
 
 func (p *Picker) Apply(msg message.Msg) (message.Msg, error) {
-	log.With("msg", msg).Infof("picking...")
+	log.With("msg", msg).Debugf("picking...")
```

Comment:
since it wouldn't seem obvious, you can change this to `Debugln` (same below as well)